### ISSUE TITLE
"institution not found" flag

### ISSUE THREAD

**Issue by fabcpinho** [plaid/plaid-link-android#7]
Hey guys, i used to use plaid with a webview on my project and I had an "institution_not_found" status flag that was send when the user search for a non existing organization and then click the add manually button.
I am trying to integrate your SDK but when I click that button I land on the onCancelled callback without any context data. 

Can you help me with this?

**Comment by zsweigart** [comment]
The onCancelled callback is for when a user explicitly closes link and it only has the session id and institution information if the user selected one.  If the user has not selected one then there is no information to provide.

You can however, use a `LinkEventListener`.  Using version 0.2.0 or greater you'll add it via:
```
Plaid.setLinkEventListener(LinkEventListener {
      Log.i(TAG, it.eventName)
    })
```

Here you will get all the events you used to receive before.  So you could keep track of the last event emitted and if it's an `institution_not_found` event you can respond accordingly.

**Comment by zsweigart** [comment]
I just went through the flow and didn't see the `institution_not_found` event so I've flagged with our backend team.  I will keep you posted

**Comment by zsweigart** [comment]
This is an issue with the sdk, we should be passing this back in the cancellation event we will add metadata in the next release which is scheduled for next week.  Keep a lookout for the update.

**Comment by zsweigart** [comment]
This has now been fixed in 0.3.0.   The `LinkCancellation` object now has an exit_status field which will contain this value.
